programs/mango-v4/src/logs.rs

```diff
@@ -803,6 +803,7 @@ pub struct TokenCollateralFeeLog {
     pub token_index: u16,
     pub asset_usage_fraction: i128,
     pub fee: i128,
+    pub price: i128,
```

Comment:
are you happy with changing the event definition like this? since v0.23 isn't released yet I'd say it's ok if it doesn't make things hard on the scraping side

Reply:
yes please, this makes my life a lot easier